### PR TITLE
Issues/380

### LINKED ISSUE #380: Merge master into dev
Merge master into dev so that the eventual dev->master merge will be clean.

### PR DESCRIPTION
**Pull request by noahl**
Merge master into dev in preparation for merging dev into master.

Closes #380 .

### CODE REVIEW

File: rho.spec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Name: rho
 Version: 0.0.29
-Release: 1%{?dist}
+Release: 5%{?dist}
```
Comment (chambridge) [inline rho.spec]: Should keep release at 1 now we are bumping the version.

File: rho.spec [hunk 2]
```diff
@@ -66,7 +68,7 @@ rm -rf $RPM_BUILD_ROOT
 * Tue Oct 17 2017 Noah Lavine <nlavine@redhat.com> 0.0.29
```
Comment (chambridge) [inline rho.spec]: Should be 0.0.29-1

**Comment by noahl** [comment]
@chambridge I pushed a new commit that I think fixes the problems you found. Please take a look.

**Review by chambridge — APPROVED** [review]
👍 Super!!

**Pull request merged**